Blish HUD/Controls/Menu.cs

```diff
@@ -128,24 +124,24 @@ public class Menu : Container, IMenuItem {
         }
 
         public override void UpdateContainer(GameTime gameTime) {
-            int totalItemHeight = _children.Where(c => c.Visible).Max(c => c.Bottom);
+            int totalItemHeight = 0;
+
+            for (int i = 0; i < _children.Count; i++) {
+                totalItemHeight = Math.Max(_children[i].Bottom, totalItemHeight);
+            }
```

Comment:
Why is this better than

```cs
var totalItemHeight = _children.Max(c => c.Bottom);
```
?

Reply:
Saves the alloc from `GetEnumerator()` and from the LINQ call. I'm looking to reduce LINQ + foreach usage where I find it in the update or render loop.  One-offs are fine (like action from clicking a button or something).

Not a huge issue, but its 2 seconds switching between implementations with ReSharper.  I'll benchmark allocs and performance later and see if I can back these actions with numbers.  I'm mostly basing this on (primarily for Unity) performance write ups I've read.

Reply:
Here, got tests:

``` ini

BenchmarkDotNet=v0.11.5, OS=Windows 10.0.14393.447 (1607/AnniversaryUpdate/Redstone1), VM=VirtualBox
Intel Core i7-8550U CPU 1.80GHz (Kaby Lake R), 1 CPU, 1 logical core and 1 physical core
Frequency=1945297 Hz, Resolution=514.0603 ns, Timer=TSC
  [Host]     : .NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.7.3062.0
  DefaultJob : .NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.7.3062.0


```
|      Method |        Mean |      Error |     StdDev |      Median | Rank |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|------------ |------------:|-----------:|-----------:|------------:|-----:|-------:|------:|------:|----------:|
|    WithLinq | 3,649.66 ns | 109.791 ns | 309.667 ns | 3,553.09 ns |    2 | 0.0153 |     - |     - |      80 B |
| WithoutLinq |    71.29 ns |   1.668 ns |   4.423 ns |    69.81 ns |    1 |      - |     - |     - |         - |